lib/3llo/utils.rb

```diff
@@ -17,6 +17,8 @@ module Utils
       "sky" => 36
     }.freeze()
 
+    TRELLO_LABEL_COLOR = %w[red pink blue green purple yellow orange sky].freeze()
```

Comment:
You can use `Hash#keys`

Reply:
There are some unnecessary keys Trello doesn't support as cyan and lime. It's not really friendly when you let user choose and return the unexpected response.

Reply:
ok that makes sense.